python/test/unit/fem/test_symmetric_tensor.py

```diff
@@ -0,0 +1,23 @@
+from mpi4py import MPI
+
+import numpy as np
+import pytest
+
+import basix.ufl
+import dolfinx
+import ufl
+
+
+@pytest.mark.parametrize("degree", range(1, 4))
+def test_transpose(degree):
+    mesh = dolfinx.mesh.create_unit_square(MPI.COMM_WORLD, 10, 10)
+    for symmetry in [True, False]:
```

Comment:
You may want to move this to `pytest.mark.parametrize`, so that if only one of the two cases fails in future it will be more clear which one of the two cases is failing.

Reply:
Good idea